### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by dyrtool**
fixed a typo in pwncat --help

### CODE REVIEW

**Review by cytopia — COMMENTED** [review]
@dyrtool 
Thanks for the contribution.

Could you also adjust it accordingly in 

* README.md
* docs/index.html


As well as run `make docs` to generate the documentation accordingly.

**Comment by dyrtool** [comment]
Unfortunately i couldn't get the `make docs` command to work (it gave me the error "help2man: couldn't get --help info from bin/pwncat") so i just fixed all other occurences of the typo

File: docs/pwncat.api.html [hunk 1]
```diff
@@ -18111,4 +18111,4 @@ <h4><code><a title="pwncat.TransformSafeword" href="#pwncat.TransformSafeword">T
 <p>Generated by <a href="https://pdoc3.github.io/pdoc"><cite>pdoc</cite> 0.9.2</a>.</p>
 </footer>
 </body>
-</html>
\ No newline at end of file
+</html>
```
Comment (cytopia) [inline docs/pwncat.api.html:18114]: There's the trailing newline too much. I've tried to commit this via GitHub UI, but the commit does not seem to remove it

Comment (dyrtool) [reply docs/pwncat.api.html]: Yep, should be fixed now

Comment (cytopia) [reply docs/pwncat.api.html]: Looks all good to me. I will let CI figure out the rest.

**Review by cytopia — APPROVED** [review]
:+1:

**Pull request merged**